src/test/java/edu/ucsb/ucsbcslas/models/TutorAssignmentOfficeHourViewTest.java

```diff
@@ -0,0 +1,66 @@
+    assertNotEquals(tutorAssignmentOfficeHourView, new Object());
+  }
+
+  @Test
+  public void testTutorAssignmentOfficeHourView_equalsSelf() throws Exception {
+    Course c = new Course(1L, "course 1", "F20", "fname", "lname", "email");
+    Tutor t = new Tutor(1L, "Seth", "VanB", "vanbrocklin@ucsb.edu");
+    TutorAssignment expectedTutorAssignments = new TutorAssignment(1L, c, t, "TA");
+    List<OnlineOfficeHours> expectedOnlineOfficeHoursList = new ArrayList<>();
+    OnlineOfficeHours onlineOfficeHours_1 = new OnlineOfficeHours(1L, expectedTutorAssignments, "Tuesday", "4:00 PM", "6:00 PM", "https://ucsb.zoom.us/j/92714889391?pwd=dXlaVTFUS1NuNm5xL0NMUUFjRDVndz09", "Scott closes the room early sometimes but he will still be on slack!");
+    expectedOnlineOfficeHoursList.add(onlineOfficeHours_1);
+    TutorAssignmentOfficeHourView tutorAssignmentOfficeHourView = new TutorAssignmentOfficeHourView(expectedTutorAssignments,expectedOnlineOfficeHoursList);
+    assertEquals(tutorAssignmentOfficeHourView, tutorAssignmentOfficeHourView);
+  }
+
+  @Test
+  public void testTutorAssignmentOfficeHourView_toString() throws Exception {
+    Course c = new Course(1L, "course 1", "F20", "fname", "lname", "email");
+    Tutor t = new Tutor(1L, "Seth", "VanB", "vanbrocklin@ucsb.edu");
+    TutorAssignment expectedTutorAssignments = new TutorAssignment(1L, c, t, "TA");
+    List<OnlineOfficeHours> expectedOnlineOfficeHoursList = new ArrayList<>();
+    OnlineOfficeHours onlineOfficeHours_1 = new OnlineOfficeHours(1L, expectedTutorAssignments, "Tuesday", "4:00 PM", "6:00 PM", "https://ucsb.zoom.us/j/92714889391?pwd=dXlaVTFUS1NuNm5xL0NMUUFjRDVndz09", "Scott closes the room early sometimes but he will still be on slack!");
+    expectedOnlineOfficeHoursList.add(onlineOfficeHours_1);
+    TutorAssignmentOfficeHourView tutorAssignmentOfficeHourView = new TutorAssignmentOfficeHourView(expectedTutorAssignments,expectedOnlineOfficeHoursList);
+    String tutorAssignmentExpected = "{ id='1', course='Course[ id=1, name=course 1, quarter=F20, instructorFirstName=fname, instructorLastName=lname, instructorEmail=email ]', tutor='{ id='1', firstName='Seth', lastName='VanB', email='vanbrocklin@ucsb.edu'}', assignmentType='TA'},";
+    String onlineOfficeHourExpected = "{ id='1', tutorAssignment='{ id='1', course='Course[ id=1, name=course 1, quarter=F20, instructorFirstName=fname, instructorLastName=lname, instructorEmail=email ]', tutor='{ id='1', firstName='Seth', lastName='VanB', email='vanbrocklin@ucsb.edu'}', assignmentType='TA'}', dayOfWeek='Tuesday', startTime='4:00 PM', endTime='6:00 PM', zoomRoomLink='https://ucsb.zoom.us/j/92714889391?pwd=dXlaVTFUS1NuNm5xL0NMUUFjRDVndz09', notes='Scott closes the room early sometimes but he will still be on slack!'}";
+    assertEquals(tutorAssignmentOfficeHourView.toString(), "[tutorAssignment="+tutorAssignmentExpected+" onlineOfficeHours=["+ onlineOfficeHourExpected+"]]" );
+  }
+}
```

Comment:
This is not a full review, but I noticed something that is a big red flag: you committed actual class and student information into the public. Please remove this ASAP.

@pconrad We might have to take some additional precautions to make sure others can't see this information, especially since this is a public repository.

Reply:
I don't quite understand what do you mean by sharing information with the public since the file you are talking about here is a test file.

Reply:
And also how was student info shown to the public since we did not write anything related to users’ roles over here?

Reply:
Andrew is referring to line 44 and 56 where you use actual student information as a test object. Here you should be using values like John Doe or test@test.com. This repository is public and as such all of the files can be accessed by anyone.

Reply:
Hi TA, I have solved the required changes. Please double check.

Reply:
The Zoom links are still present. Can we remove those as well?

Reply:
Actual zoom link has been removed.